### ISSUE TITLE
[Android][OpenGL]Pursuit Force not enough FPS performance (ULJM05097)

### ISSUE THREAD

**Issue by nassau-tk** [hrydgard/ppsspp#12335]
What happens?
There is no enough FPS. (It happens certainly by more than one specific situation.)

What should happen?
More smooth work.

What hardware, operating system, and PPSSPP version?
On desktop, GPU matters for graphical issues.
PPSSPP Ver. : 1.8.0-617
OS: Android 9.0
DeviceName: SONY SO-02K / Xperia XZ1 Compact (Japan)
CPU: Snapdragon835 (Octa-core)
GPU: Adreno 540
Resolution: 720×1280
RAM: 4GB
ROM:32GB
ExternalStrage: microSDXC 512GB

Please check the attached dump file.
[recording_ULJM05097.zip](https://github.com/hrydgard/ppsspp/files/3623915/recording_ULJM05097.zip)

**Comment by Panderner** [comment]
Are you using opengl or vulkan?

**Comment by nassau-tk** [comment]
I'm using and talking about OpenGL.
But....Behavior depends on a stage.（OpenGL & Vulkan)
example1)
1st mission of (WARLORDS) : OpenGL is bad FPS.
1st mission of (WARLORDS) : Vulkan is Good FPS.
example2)
1st mission of (Capelli Family) : OpenGL is bad FPS.
1st mission of (Capelli Family) : Vulkan is bad FPS.

There are a lot of situations a problem generates,
 but OPENGL is confirming that even Vulkan happens.

**Comment by marosis** [comment]
.log file is needed

**Comment by hrydgard** [comment]
Not really, it's pretty well known that these games use bezier patch tesselation for the road rendering which is a bit expensive. Enabling the option "Hardware tesselation" may help improve performance.

**Comment by nassau-tk** [comment]
>>marosis
I apologize for that by an elementary question.
How can I get the .log file?
Please tell me.

>>hrydgard
Sorry, This issue happens by "Hardware tesselation" turn on.

But..."Hardware tesselation" turn off.
Result is below.
1st mission of (WARLORDS) : OpenGL is **Good** FPS.
1st mission of (Capelli Family) : OpenGL is bad FPS.

The person who turned off was smooth drawing at one stage.

**Comment by unknownbrackets** [comment]
If hardware tessellation doesn't help make it fast (which I think can make it slower on some GL devices), and setting the curve quality to low doesn't help, there's probably not a lot else we can do.

This game used a fairly advanced rendering method on the PSP, and it takes more power to emulate.

-[Unknown]

**Comment by nassau-tk** [comment]
@unknownbrackets

I understand.
PPSSPP expects to become better and waits!

**Comment by nassau-tk** [comment]
Version : 1.9.3-64
A problem of FPS seems settled.
The once and Close.

**Comment by nassau-tk** [comment]
um...

It's will not happen on Galaxy S20.(SD86g+RAM12GB)

**Comment by hrydgard** [comment]
will not happen?

**Comment by nassau-tk** [comment]
Yes, I tested same scene in the game.
But, It's not happen.
(It's means "Not goes down FPS when Timing of Playing voice".)
Is this a reason by machine spec?

Notice : Offcourse, Xperia XZ1 compact (D835+RAM 4GB) is happen yet.

**Comment by Gamemulatorer** [comment]
Duplicate of https://github.com/hrydgard/ppsspp/issues/11216#issuecomment-925107278?

**Comment by Panderner** [comment]
> Duplicate of [#11216 (comment)](https://github.com/hrydgard/ppsspp/issues/11216#issuecomment-925107278)?

Umm it's a performance issue

**Comment by Gamemulatorer** [comment]
> > Duplicate of [#11216 (comment)](https://github.com/hrydgard/ppsspp/issues/11216#issuecomment-925107278)?
> 
> Umm it's a performance issue

Still related read my comment there.

**Comment by nassau-tk** [comment]
> > > Duplicate of [#11216 (comment)](https://github.com/hrydgard/ppsspp/issues/11216#issuecomment-925107278)?
> > 
> > 
> > Umm it's a performance issue
> 
> Still related read my comment there.

May be not related I think.
